{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,111 +1,160 @@
+  - `(2)`: your main task as a chef author is to implement all the extraction
+    and content transformation described in the spec. If you run into any kind
+    of difficulties with this step post a question in the LE slack channel
+    `#sushi-chefs` and someone from the content team will be able assist you.
+    For example, "Hello @here I'm having trouble with the {{cookiecutter.channel_name}} chef
+    because X and Y cannot be organized according to the spec because Z."
+    For complicated or very large channels the spec may go through multiple iterations.
+  - `(3)`: once the channel is on Studio you can preview the structure there
+    and create or update a notion card with the channel information.
+    The next step is to export the channel in the format necessary for use in
+    Kolibri using the `PUBLISH` button on Studio. The PUBLISH action exports
+    all the channel metadata to a sqlite3 DB file
+    `https://studio.learningequality.org/content/databases/{channel_id}.sqlite3`
+    the first time a channel is PUBLISH-ed a secret token is generated that can
+    be used to import the channel in Kolibri. Note down the channel's token.
+  - `(4)`: the next step is to import your channel into a Kolibri instance. You
+    can use Kolibri installed on your local machine or an online demo server.
+    Admin (`devowner`) credentials for the demo server will be provided for you
+    so that you can import and update the channel every time you push a new version.
+    Follow these steps to import your channel `Device` > `IMPORT` > `ONLINE` >
+    `Try adding a token`, add the channel token, select all nodes > `IMPORT`.
+  - `(5)`: You can now go to the Kolibri Learn tab and preview your channel to
+    see it the way learners will see it. Take the time to click around and browse
+    the content to make sure everything works as expected. Update the notion card
+    and leave a comment. For example "First draft of channel uploaded to demo server."
+    This would be a good time to ask a member of the LE content team to review
+    the channel. You can do this using the `@Person Name` in your notion comment.
+    Consult the content source notion card to know who the relevant people to tag.
+    For example, you can @-comment the `Library` person on the card to ask them
+    to review the channel—be sure to specify the channel's "level of readiness"
+    in your comment, e.g., if it's a draft version for initial feedback, or
+    the near-final, spec-compliant version ready for detailed review and QA.
+    For async technical questions tag the `SushOps` person on the card or post
+    your question in the `#sushi-chefs` channel. For example, "I downloaded this
+    html and js content, but it doesn't render right in Kolibri because of the
+    iframe sandboxing." or "Does anyone have sample code for extracting content
+    X from a shared drive link Y of type Z?".
```

Comment:
@kollivier This is copy-pasted from the ricecooker docs
https://ricecooker.readthedocs.io/en/latest/platform/content_workflows.html

I thought it would be useful to have a copy right in the README as well so chef authors will have Rubric and procedures right there in the README. It's duplication but might keep down on the onboarding.

The other option would be to link to the ricecooker docs page (easier to maintain) but not likely that chef author will click that link on their own...

Reply:
I think it's better to have the link, mainly because if we update the docs and/or change things, it likely won't be updated in the repos. I don't think most people will read these more than once, so the case I'm worried about is someone picking up this chef a year or years later and reading up on an outdated process.

Reply:
Okay, I left just the diagram and put link to ricecooker docs instead.